scripts/ghostbuster.js

```diff
@@ -86,9 +86,9 @@ function getAllPackageJsons() {
  * @param {Set<string>} users
  */
 async function fetchGhosts(users) {
-    console.log("Checking for deleted accounts...");
+    console.error("Checking for deleted accounts...");
     const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
-    const maxPageSize = 2000;
+    const maxPageSize = 500;
```

Comment:
Drive-by, but 2000 consistently caused GitHub to 503 on my machine. It always works in CI, though.